src/Components/ServiceScene/Breakdowns/AddToFiltersButton.tsx

```diff
@@ -103,7 +104,12 @@ function resolveVariableNameForField(field: string, scene: SceneObject) {
 
 export class AddToFiltersButton extends SceneObjectBase<AddToFiltersButtonState> {
   public onClick = (type: FilterType) => {
-    const filter = getFilter(this.state.frame);
+    let filter = getFilter(this.state.frame);
+
+    if (this.state.labelName && !filter) {
+      filter = { name: this.state.labelName, value: '""' };
+    }
+
```

Comment:
So we get this behavior when there's a label name but no frame name. If we could specify from the "outside" if the filter should include/exclude all, it would be ideal.